packages/worker/src/main.ts

```diff
@@ -38,7 +39,52 @@ Amplify.configure(
   }
 );
 
+class ConverseSessionTracker {
+  private sessions: { isFinished: boolean; cancellationToken: CancellationToken }[] = [];
```

Comment:
This might be overkill, because keeping the only last session is usually enough.  However, we'd like to keep this implementation because we might add a featrure to allow running multiple concurrent converse sessions in the near future.